plugin/pkg/admission/imagepolicy/admission.go

```diff
@@ -181,6 +199,12 @@ func (a *Plugin) admitPod(pod *api.Pod, attributes admission.Attributes, review
 		return errors.New("one or more images rejected by webhook backend")
 	}
 
+	if review.Status.AttributeAnnotations == nil {
+		return nil
+	}
+	for k, v := range review.Status.AttributeAnnotations {
+		attributes.AddAnnotation(AuditKeyPrefix+k, v)
```

Comment:
check error and log a warning if if failed

Reply:
Done